server/libs/domain/src/facial-recognition/facial-recognition.repository.ts

```diff
@@ -1,9 +1,12 @@
-import { AssetFaceEntity, PersonEntity } from '@app/infra/entities';
+import { AssetEntity, AssetFaceEntity, PersonEntity } from '@app/infra/entities';
 
 export const IFacialRecognitionRepository = 'IFacialRecognitionRepository';
 
 export interface IFacialRecognitionRepository {
   createAssetFace(entity: Partial<AssetFaceEntity>): Promise<AssetFaceEntity>;
   createPerson(entity: Partial<PersonEntity>): Promise<PersonEntity>;
   savePerson(entity: Partial<PersonEntity>): Promise<PersonEntity>;
+  getById(id: string): Promise<PersonEntity | null>;
+  getAll(userId: string): Promise<PersonEntity[]>;
+  getPersonAssets(id: string): Promise<AssetEntity[]>;
```

Comment:
These all look great. Why don't we standardize by removing `person` from the names of all the methods. `create` could be for person by default, and we could maybe think about the repository being `IPersonRepository`. The machine learning interface is where the actual facial recognition calls are made.